rosa_getting_started/rosa-deleting-cluster.adoc

```diff
@@ -7,4 +7,9 @@ toc::[]
 
 Delete a {product-title} (ROSA) cluster using the `rosa` command-line.
 
+[id="prerequisites_rosa-deleting-cluster"]
+== Prerequisites
+
+* If you have a VPC created by a service, you must remove any additional network configurations or add-on services, such as VPN configurations and VPC peering, before you delete a cluster. If these configurations and services remain, the cluster does not delete properly.
```

Comment:
Three things:
1. I'd rather it state more explicitly that the managed OpenShift service (ROSA, OSD) created the service.  The way it is now, is more generic and not specific.  Like which service?  Any service that created a vpc?  So let's say, something more concrete like "If the VPC was created by the service..." or "In situations where the service created the VPC..." or "if the service was not deployed into an already existing VPC...".

We can use "{product-title}" instead of "service" to make it even clearer.

> ...you must remove any additional network configurations or add-on services, such as VPN configurations and VPC peering...

2. This give the impression that VPN and VPC peering are add-on services.  Those are example "network configurations". 

3. I would prefer to use "additional services" instead of "add-on" services, since that might lead to confusion with our add-ons. I want to to be clear that we mean other **AWS services** that were added to the VPC.